libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/InstanceView/InstanceView.tsx

```diff
@@ -357,4 +367,52 @@ export class InstanceView extends React.Component<
     };
     this.setState(reloadDataFunc);
   }
+
+  private onRenderLabel = (type: SelectionType) => (
+    p: IChoiceGroupOption | undefined
+  ) => {
+    const classNames = InstanceViewStyles();
+    let selectionText = "";
+    switch (type) {
+      case SelectionType.AllSelectionType:
+        selectionText =
+          this.state.selectionDetails.selectedAllSelectedIndexes.length +
+          " selected";
```

Comment:
we need localize all the string.
since the sequence in each language will be different, it is better to use place holder to localize whole string including number.

Reply:
done, localized